images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -284,3 +284,8 @@ function Get-PipxVersion {
     $pipxVersion = $Matches.Version
     return "Pipx $pipxVersion"
 }
+
+function Get-GraalVMVersion {
+    $version = java --version | Out-String | Select-Object -First 2 | Take-OutputPart -Part 7
```

Comment:
Does GraalVM override `java` symlink?
If it is true, we shouldn't do that.

`java` should continue to point on default Java on image